Source/Core/DolphinWX/Config/AudioConfigPane.cpp

```diff
@@ -143,7 +151,7 @@ void AudioConfigPane::OnAudioBackendChanged(wxCommandEvent& event)
 	m_volume_slider->Enable(SupportsVolumeChanges(WxStrToStr(m_audio_backend_choice->GetStringSelection())));
 	m_audio_latency_spinctrl->Enable(WxStrToStr(m_audio_backend_choice->GetStringSelection()) == BACKEND_OPENAL);
 	m_dpl2_decoder_checkbox->Enable(WxStrToStr(m_audio_backend_choice->GetStringSelection()) == BACKEND_OPENAL ||
-	                                WxStrToStr(m_audio_backend_choice->GetStringSelection()) == BACKEND_PULSEAUDIO);
```

Comment:
This looked aligned before, but that might be the GitHub web diff showing things differently...